mmv1/third_party/terraform/website/docs/r/compute_instance.html.markdown

```diff
@@ -124,7 +124,9 @@ The following arguments are supported:
 
     **Note:** If you want to update this value (resize the VM) after initial creation, you must set [`allow_stopping_for_update`](#allow_stopping_for_update) to `true`.
 
-    [Custom machine types](https://cloud.google.com/dataproc/docs/concepts/compute/custom-machine-types) can be formatted as `custom-NUMBER_OF_CPUS-AMOUNT_OF_MEMORY_MB`, e.g. `custom-6-20480` for 6 vCPU and 20GB of RAM.
+    [Custom machine types](https://cloud.google.com/dataproc/docs/concepts/compute/custom-machine-types) can be formatted as `custom-NUMBER_OF_CPUS-AMOUNT_OF_MEMORY_MB`, e.g. `custom-6-20480` for 6 vCPU and 20GB of RAM. 
+    If your custom `machine_type` is not covered by this syntax you will need to set `lifecycle_rule.ignore_changes` to `true`
```

Comment:
Do you have an example that gets mangled? I'm surprised they'd accept the input if it's not covered.

Reply:
Ok, yes i can confirm it happens on instances.

But found another buggy behavior. According to your doc the `custom-2-2048` should be `n1` but is rounded to `n2-standard-2`
![image](https://github.com/user-attachments/assets/05906371-3274-411e-a09f-6fa7efa0ef85)

Will just update the docs to warn about potential use of the `ignore_changes` because this is all over the place on API's side